### DISCUSSION TITLE
Kibana, ipphoney, logstash and map_data DOWN !

### DISCUSSION THREAD

**Discussion by Sadounet** [telekom-security/tpotce#1232] [Q&A] [answered]
Hi 
firstable, very thanks for your work it's amazing !

I have : 
vm vmware with vSAN : 4cpu 16 GB RAM my OS is Debian 11
My Cokpit working great but i can't use Kibana because i have a white page.

Version : 22.04.0

./dps.sh
![Capture d’écran 2022-12-20 à 10 51 04](https://user-images.githubusercontent.com/121093846/208637051-4ea2aa9f-5555-420b-a624-a94dab33b9ec.png)

htop
![Capture d’écran 2022-12-20 à 10 56 37](https://user-images.githubusercontent.com/121093846/208638418-719dbb9f-656c-4534-9b20-bf13a36eee50.png)

Netstat -tulpen
![Capture d’écran 2022-12-20 à 11 00 47](https://user-images.githubusercontent.com/121093846/208639687-626dca6a-8e10-4e6e-97f1-e18ff7db6118.png)

When i stop tpot Kibana is working but it's normal i don't have any datas.

systecmctl stop tpot 
<img width="1044" alt="Capture d’écran 2022-12-20 à 11 08 22" src="https://user-images.githubusercontent.com/121093846/208641409-48c11490-f1e3-4db7-8c2f-82f7e38efc7d.png">

if i systemctl start tpot i have every time a white page without dashboard and without nothing.

Can you help me ? 

best regards

**Comment by t3chn0m4g3** [comment, marked as answer]
Stop T-Pot and run `netstat -tulpen` again. It looks like a post install method installation with services interfering with T-Pot's ports.

**Comment by Sadounet** [comment]
Hi, 
I do it and finally i tried to use Netinstall on my vmware vm. You right, every time i have a conflict ports. So I installed the netiso image, and i use this command to change my virtual network for me it's was ens192 to eth0.

Now everything works perfecly fine ! Thank you for reactivity.

**Reply by t3chn0m4g3** [reply]
Awesome, thanks for the feedback 😃
